### DISCUSSION TITLE
Command Error

### DISCUSSION THREAD

**Discussion by HillFlame** [kkrypt0nn/Python-Discord-Bot-Template#100] [Q&A] [answered]
Hello, I executed the command `python3 bot.py` and received this error: `TypeError: expected token to be a str, received NoneType instead`. Could you please help?

**Comment by eclipse7723** [comment, marked as answer]
Did you followed this?

> To set up the token you will have to make use of the `.env.example` file, you should rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your actual bot's token.

Check how to get discord bot token here: https://www.writebots.com/discord-bot-token
